packages/base/src/commands/tag/__tests__/tag.test.ts

```diff
@@ -92,4 +98,59 @@ describe('execute', () => {
     expect(result.code).toBe(0)
     expect(result.context.stdout.toString()).toContain('[DRYRUN] Tag request')
   })
+
+  test('should try to determine github job display name', async () => {
+    fs.readdirSync = jest.fn().mockReturnValue([
+      {
+        name: 'Worker_2.log',
+        isFile: () => true,
+        isDirectory: () => false,
+        isBlockDevice: () => false,
+        isCharacterDevice: () => false,
+        isSymbolicLink: () => false,
+        isFIFO: () => false,
+        isSocket: () => false,
+        parentPath: '',
+        path: '',
+      },
+    ])
+    fs.readFileSync = jest.fn().mockReturnValue(`{"jobDisplayName": "real job name"}`)
+    const result = await runCLI(
+      'job',
+      ['key:value'],
+      {
+        GITHUB_ACTIONS: 'true',
+        GITHUB_SERVER_URL: 'url',
+        GITHUB_REPOSITORY: 'repo',
+        GITHUB_RUN_ID: '123',
+        GITHUB_RUN_ATTEMPT: '1',
+        GITHUB_JOB: 'fake job name',
+      },
+      ['--dry-run']
+    )
+    expect(result.code).toBe(0)
+    const out = result.context.stdout.toString()
+    expect(out).toContain('Determining github job name')
+    expect(out).toContain('"DD_GITHUB_JOB_NAME": "real job name"')
+  })
+
+  test('should not try to determine github job display name for pipelines', async () => {
+    const result = await runCLI(
+      'pipeline',
+      ['key:value'],
+      {
+        GITHUB_ACTIONS: 'true',
+        GITHUB_SERVER_URL: 'url',
+        GITHUB_REPOSITORY: 'repo',
+        GITHUB_RUN_ID: '123',
+        GITHUB_RUN_ATTEMPT: '1',
+        GITHUB_JOB: 'fake job name',
+      },
+      ['--dry-run']
+    )
+    expect(result.code).toBe(0)
+    const out = result.context.stdout.toString()
+    expect(out).not.toContain('Determining github job name')
+    expect(out).not.toContain('"DD_GITHUB_JOB_NAME": "real job name"')
```

Comment:
nitpick: a slightly better test is
```suggestion
    expect(out).not.toContain('DD_GITHUB_JOB_NAME')
```
as the `DD_GITHUB_JOB_NAME` could be a wrongly calculated value (different from `real job name`)  which would not be caught by this test